go/common/rpc/generated/enclave.proto

```diff
@@ -245,7 +245,6 @@ message AttestationReportMsg {
 
 message BlockSubmissionResponseMsg {
   ExtRollupMsg producedRollup = 1;
-  HeaderMsg rollupHead = 2;
   repeated SecretResponseMsg producedSecretResponses = 3;
```

Comment:
Don't know if it matters but these indexes need updating

Reply:
Fixed.